scss/responsive.scss

```diff
@@ -0,0 +1,62 @@
+@import 'variables';
+
+@media screen and (max-width: 700px){
+
+  .navbar-text {
+    visibility: hidden;
+  }
+
+  .card-container {
+    display: grid;
+    grid-template-columns: repeat(1, max-content);
```

Comment:
I think that by changing this grid template is enough, you can delete the rest of the class attributes :+1: